reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -2474,6 +2475,100 @@ void testConnectionClosePropagatedAsError_TE() throws Exception {
 				"POST http://%s/ HTTP/1.1\r\nHost: %s\r\nTransfer-Encoding: chunked\r\n\r\n10\r\nhello");
 	}
 
+	@Test
+	void testMatchRouteInConfiguredOrder() {
+		HttpServerRoutes serverRoutes = HttpServerRoutes.newRoutes()
+				.get("/yes/{value}", (request, response) -> response.sendString(Mono.just("/yes/{value}")))
+				.get("/yes/value", (request, response) -> response.sendString(Mono.just("/yes/value")));
+
+		DisposableServer server = HttpServer.create().handle(serverRoutes).bindNow();
```

Comment:
Please use `disposableServer` and remove `server.disposeNow()`, as `disposableServer` will be disposed with `reactor.netty.BaseHttpTest#disposeServer`